Formula/rmw.rb

```diff
@@ -6,6 +6,11 @@ class Rmw < Formula
   license "GPL-3.0-or-later"
   head "https://github.com/theimpossibleastronaut/rmw.git"
 
+  livecheck do
+    url :stable
+    strategy :github_latest
```

Comment:
```suggestion
    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

The standard regex for Git tags like `1.2.3`/`v1.2.3` also gives `0.7.09` as the latest version, so using the `GithubLatest` strategy isn't necessary here. If the standard regex becomes insufficient for this in the future, we can modify it at that time to restrict the first number to something less than four digits (e.g., `/^v?(\d{1,3}(?:\.\d+)+)$/i`). `GithubLatest` shouldn't be necessary for this formula until we truly can't obtain the latest version using the `Git` strategy.